### PR TITLE
chore: release v0.2.0

### PR DESCRIPTION
**Pull request by zvolin**
## 🤖 New release
* `blockstore`: 0.1.1 -> 0.2.0 (⚠️ API breaking changes)

### ⚠️ `blockstore` breaking changes

```
--- failure auto_trait_impl_removed: auto trait no longer implemented ---

Description:
A public type has stopped implementing one or more auto traits. This can break downstream code that depends on the traits being implemented.
        ref: https://doc.rust-lang.org/reference/special-types-and-traits.html#auto-traits
       impl: https://github.com/obi1kenobi/cargo-semver-checks/tree/v0.30.0/src/lints/auto_trait_impl_removed.ron

Failed in:
  type BlockstoreError is no longer UnwindSafe, in /tmp/.tmpIQmsp1/blockstore/src/lib.rs:33
  type BlockstoreError is no longer RefUnwindSafe, in /tmp/.tmpIQmsp1/blockstore/src/lib.rs:33

--- failure derive_trait_impl_removed: built-in derived trait no longer implemented ---

Description:
A public type has stopped deriving one or more traits. This can break downstream code that depends on those types implementing those traits.
        ref: https://doc.rust-lang.org/reference/attributes/derive.html#derive
       impl: https://github.com/obi1kenobi/cargo-semver-checks/tree/v0.30.0/src/lints/derive_trait_impl_removed.ron

Failed in:
  type BlockstoreError no longer derives PartialEq, in /tmp/.tmpIQmsp1/blockstore/src/lib.rs:33

--- failure enum_variant_added: enum variant added on exhaustive enum ---

Description:
A publicly-visible enum without #[non_exhaustive] has a new variant.
        ref: https://doc.rust-lang.org/cargo/reference/semver.html#enum-variant-new
       impl: https://github.com/obi1kenobi/cargo-semver-checks/tree/v0.30.0/src/lints/enum_variant_added.ron

Failed in:
  variant BlockstoreError:IoError in /tmp/.tmpIQmsp1/blockstore/src/lib.rs:44
  variant BlockstoreError:StorageCorrupted in /tmp/.tmpIQmsp1/blockstore/src/lib.rs:48
  variant BlockstoreError:BackingStoreError in /tmp/.tmpIQmsp1/blockstore/src/lib.rs:52

--- failure enum_variant_missing: pub enum variant removed or renamed ---

Description:
A publicly-visible enum has at least one variant that is no longer available under its prior name. It may have been renamed or removed entirely.
        ref: https://doc.rust-lang.org/cargo/reference/semver.html#item-remove
       impl: https://github.com/obi1kenobi/cargo-semver-checks/tree/v0.30.0/src/lints/enum_variant_missing.ron

Failed in:
  variant BlockstoreError::CidExists, previously in file /tmp/.tmpdwMFzF/blockstore/src/lib.rs:19
```

<details><summary><i><b>Changelog</b></i></summary><p>

<blockquote>

### Added
- feat!(node): Implement DASer ([#223](https://github.com/eigerco/blockstore/pull/223))
- *(blockstore)* add IndexedDb blockstore ([#221](https://github.com/eigerco/blockstore/pull/221))
- feat!(blockstore): add sled blockstore ([#217](https://github.com/eigerco/blockstore/pull/217))
- *(blockstore)* Implement LruBlockstore ([#207](https://github.com/eigerco/blockstore/pull/207))

### Fixed
- fix!(blockstore): remove an error if cid already exists ([#224](https://github.com/eigerco/blockstore/pull/224))

### Other
- *(ci)* add gitignore and CI workflows
</blockquote>

</p></details>

---
This PR was generated with [release-plz](https://github.com/MarcoIeni/release-plz/).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0](https://github.com/eigerco/blockstore/compare/v0.1.1...v0.2.0) - 2024-03-25
+
+### Added
+- feat!(node): Implement DASer ([#223](https://github.com/eigerco/blockstore/pull/223))
```
Comment (zvolin) [inline CHANGELOG.md:12]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0](https://github.com/eigerco/blockstore/compare/v0.1.1...v0.2.0) - 2024-03-25
+
+### Added
+- *(blockstore)* add IndexedDb blockstore ([#221](https://github.com/eigerco/blockstore/pull/221))
```
Comment (oblique) [inline CHANGELOG.md:12]: Manually change all the links to `[eigerco/lumina#XXX](https://github.com/eigerco/lumina/pull/XXX)`

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- feat!(blockstore): add sled blockstore ([#217](https://github.com/eigerco/blockstore/pull/217))
+- *(blockstore)* Implement LruBlockstore ([#207](https://github.com/eigerco/blockstore/pull/207))
+
+### Fixed
+- fix!(blockstore): remove an error if cid already exists ([#224](https://github.com/eigerco/blockstore/pull/224))
```
Comment (zvolin) [inline CHANGELOG.md:17]:
```suggestion
- *(blockstore)* add IndexedDb blockstore ([#221](https://github.com/eigerco/lumina/pull/221))
- feat!(blockstore): add sled blockstore ([#217](https://github.com/eigerco/lumina/pull/217))
- *(blockstore)* Implement LruBlockstore ([#207](https://github.com/eigerco/lumina/pull/207))

### Fixed
- fix!(blockstore): remove an error if cid already exists ([#224](https://github.com/eigerco/lumina/pull/224))
```

**Review by oblique — APPROVED** [review]

**Review by fl0rek — APPROVED** [review]
⚙️

**Pull request merged**